src/expenses/expenses.router.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const express = require('express');
+const expensesRouter = express.Router();
+const { expensesController } = require('./expenses.controller');
+
+expensesRouter.get('/expenses', expensesController.getAllExpenses);
+
+expensesRouter.post('/expenses', expensesController.addNextExpense);
+
+expensesRouter.delete('/expenses/:expenseId', expensesController.deleteExpense);
+
+expensesRouter.get('/expenses/:expenseId', expensesController.getOneExpense);
+
+expensesRouter.put('/expenses/:expenseId', expensesController.expenseToUpdate);
```

Comment:
please checkout my suggestion in `src/createServer.js`
```suggestion
expensesRouter.get('/', expensesController.getAllExpenses);

expensesRouter.post('/', expensesController.addNextExpense);

expensesRouter.delete('/:expenseId', expensesController.deleteExpense);

expensesRouter.get('/:expenseId', expensesController.getOneExpense);

expensesRouter.put('/:expenseId', expensesController.expenseToUpdate);
```